bundles/org.openhab.binding.electroluxair/README.md

```diff
@@ -0,0 +1,111 @@
+# ElectroluxAir Binding
+
+This is an openHAB binding for the Pure A9 Air Purifier, by Electrolux.
+
+This binding uses the Electrolux Delta REST API.
+
+![Electrolux Pure A9](doc/electrolux_pure_a9.webp)
```

Comment:
@Confectrician Will this render correctly in the final documentation?

Reply:
Sorry for the late answer.

No. We only support png files at the moment.